Python/books.md

```diff
@@ -1,6 +1,18 @@
-Name | Link
------------- | ------------- 
-Learn Python the Hard Way | http://learnpythonthehardway.org/
-Core Python Programming (2nd Edition) | http://www.amazon.com/Core-Python-Programming-Wesley-Chun/dp/0132269937
-How to Think Like a Computer Scientist: Learning with Python | http://www.lulu.com/shop/allen-downey/how-to-think-like-a-computer-scientist-learning-with-python/paperback/product-3487688.html
-Violent Python | http://shop.oreilly.com/product/9781597499576.do
+### Python Books
+
+* [Learn Python the Hard Way ](http://learnpythonthehardway.org/) - Exercise based step by step guide to basic python programming.
+* [Core Python Programming (2nd Edition)](http://www.amazon.com/Core-Python-Programming-Wesley-Chun/dp/0132269937) - Simple guide on core python concepts like Variable, Loops, Collections etc.
+* [How to Think Like a Computer Scientist: Learning with Python](http://www.lulu.com/shop/allen-downey/how-to-think-like-a-computer-scientist-learning-with-python/paperback/product-3487688.html) - Here you'll find a short introduction to Logo as a general purpose programming language, as opposed to its limited educational use with turtle graphics.
+* [Violent Python ](http://shop.oreilly.com/product/9781597499576.do) - A Cookbook for Hackers, Forensic Analysts, Penetration Testers and Security Engineers.
+* [Core python application programming](https://5d0c00d0babedc9d7abd62f30361d865b480e494.googledrive.com/host/0B4m99Zn8K7eGZzlUTFZCX20yVE0/python/Core%20Python%20Applications%20Programming%203rd.pdf) - Mostly based on Generel programming concept and Web Development.

```

Comment:
this is a Google drive URL and not acceptable! Please share an original publisher URL.